content/en/network_monitoring/devices/netflow.md

```diff
@@ -8,17 +8,17 @@ further_reading:
   text: "Using Profiles with Network Device Monitoring"
 ---
 
-<div class="alert alert-warning">NetFlow Monitoring for Datadog Network Device Monitoring is in private beta.</div>
+<div class="alert alert-warning">NetFlow Monitoring for Datadog Network Device Monitoring is in public beta.</div>
```

Comment:
Just confirming on this part, since your PR description says to list it as private beta and you're switching it to public here?

Reply:
Yes, goal is to switch it from saying private to public